### PR TITLE
To do list - React JS

### PR DESCRIPTION
**Pull request by RitobrotoMukherjee**
## Review Changes:
- Updated site and added `React Router Dom`
- Developed **Author**, **About Us**, and **404** pages.

## Developed To Do List
- Used `CRA`
- Used `class component` and `functional component` both for practice
- Did basic styling for the wireframe
- Used `CSS flexbox`
- `media queries` not implemented
- Saved data in `localStorage`
- `React Router` is not implemented. 
- Followed `Parent - Child Layout` based architecture
- Followed `DRY`, `KISS` and `YAGNI`

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+import './index.css';
+import App from './App';
+import reportWebVitals from './reportWebVitals';
+
+const root = ReactDOM.createRoot(document.getElementById('to-do-app'));
+root.render(
+  <React.StrictMode>
+    <App />
+  </React.StrictMode>,
+);
```
Comment (Chrissiku) [inline src/index.js]: - [x] Kindly ensure you implement the routing as listed in the project instructions. You can read. You can use the [ given link](https://ibaslogic.com/routing-with-react-router/), which will help you implement that

### Instruction given instruction
routing

**Review by Chrissiku — CHANGES_REQUESTED** [review]
Hi @RitobrotoMukherjee  , 

## Status : Request Changes  

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!

## Higlights 👍 
- No linters errors ✔️ 
- Use of gitflow ✔️ 
- Use of github workflow ✔️ 
-  No linters errors ✔️ 
-  Use of React components and Hooks✔️

## Suggested changes ♻️ 

- [x] Using the enter key to add new to-dos works correctly but it would be better to also add a button, as shown on the image bellow,  for a better user experience and accessibility
changes


## _Check Also the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Comment by RitobrotoMukherjee** [comment]
Hi @Chrissiku,

Thanks for the review and the suggestions. Will implement ASAP.

**Review by Kimutai01 — APPROVED** [review]
Hi @RitobrotoMukherjee  :wave: 

Your project is complete! There is nothing else to say other than... it's time to merge it :shipit: 
**To highlight**
:heavy_check_mark:  No linters errors
:heavy_check_mark:  The code follows DRY, KISS and YAGNI
Congratulations! :tada: 

Cheers and Happy coding! :clap: :clap: 
unnamed


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Comment by RitobrotoMukherjee** [comment]
Hi @Kimutai01,

Thanks for approving my project. ⏩

**Pull request merged**